packages/grafana-data/src/types/logs.ts

```diff
@@ -136,8 +136,16 @@ export interface DataSourceWithLogsContextSupport<TQuery extends DataQuery = Dat
 
   /**
    * Retrieve the context query object for a given log row. This is currently used to open LogContext queries in a split view.
+   * The `cacheFilters` parameter can be used to force a refetch of the cached applied filters. Default value `true`.
+   * @alpha
+   * @internal
    */
-  getLogRowContextQuery?: (row: LogRowModel, options?: LogRowContextOptions, query?: TQuery) => Promise<TQuery | null>;
+  getLogRowContextQuery?: (
```

Comment:
I don't think this should be alpha/internal as we already have it in documentation https://grafana.com/developers/plugin-tools/tutorials/build-a-logs-data-source-plugin. So I would remove the `@alpha/@internal`. Or at least `@internal`. And also that documentation needs to be updated.